dependencies/job-config.json

```
@@ -523,15 +523,15 @@
       },
       "3.6": {
         "CSV_VERSION": "3.6.0",
-        "CSV_VERSION_PREV": "3.5.0"
+        "CSV_VERSION_PREV": "3.5.0-0.1679521107.p"
       },
       "3.7": {
         "CSV_VERSION": "3.7.0",
-        "CSV_VERSION_PREV": "3.5.0"
+        "CSV_VERSION_PREV": "3.6.0"
       },
       "3.x": {
         "CSV_VERSION": "3.7.0",
-        "CSV_VERSION_PREV": "3.5.0"
+        "CSV_VERSION_PREV": "3.6.0"
```

There is no 3.6.0, so this should be the last released version

```suggestion
        "CSV_VERSION_PREV": "3.5.0-0.1679521107.p"
```